docs/components/receive.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Receiver
+type: docs
+menu: components
+---
+
+# Receiver
+
+The `thanos receive` command implements the [Prometheus Remote Write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write). It builds on top of existing Prometheus TSDB and retains their usefulness while extending their functionality with long-term-storage, horizontal scalability, and downsampling. It exposes the StoreAPI so that [Thanos Queriers](./query.md) can query received metrics in real-time. The [Thanos Sidecar](./sidecar.md) is not sufficient for this, as the system would always lag the block length behind (typically 2 hours).
+
+We recommend this component to users who can only push into a Thanos due to air-gapped environments. Please note the various pros and cons of pushing metrics.
```

Comment:
```suggestion
We recommend this component to users who can only push into a Thanos due to air-gapped, or egress only environments. Please note the various pros and cons of pushing metrics.
```

Otherwise well put :muscle: